src/loaders/google_map_loader.js

```diff
@@ -50,23 +54,21 @@ export default (bootstrapURLKeys, heatmapLibrary) => {
     throw new Error('google map cannot be loaded outside browser env');
   }
 
-  if (!loader_) {
```

Comment:
@jpoehnelt I feel there's no need to check for this, that doesn't change much, because loader will load another script anyways.